twatbot.py

```diff
@@ -150,6 +153,8 @@ def add_owner_to_list(user, lst):
 def html_header():
 	header = """<!DOCTYPE html><html><head>
 <title>%s</title><meta charset="utf-8"/>""" % args.title
+	## check user box
+	header += '<script src="js/searchbar_check.js"></script>'
```

Comment:
can we just inline the js in a <script tag> here ? the --dir mode requires that we copy any filesystem files around, which is a bit annoying